File: dataset/CoT/GEMINI_gemini-2.5-pro/Task35.java
import java.util.ArrayList;
import java.util.Arrays;

class Task35 {
    public static int countPassengers(ArrayList<int[]> stops) {
        int passengers = 0;
        for (int[] stop : stops) {
            passengers += stop[0];
            passengers -= stop[1];
        }
        return passengers;
    }

    public static void main(String[] args) {
        // Test Case 1
        ArrayList<int[]> test1 = new ArrayList<>(Arrays.asList(new int[]{10, 0}, new int[]{3, 5}, new int[]{5, 8}));
        System.out.println("Remaining passengers: " + countPassengers(test1)); // Expected: 5

        // Test Case 2
        ArrayList<int[]> test2 = new ArrayList<>(Arrays.asList(new int[]{3, 0}, new int[]{9, 1}, new int[]{4, 10}, new int[]{12, 2}, new int[]{6, 1}, new int[]{7, 10}));
        System.out.println("Remaining passengers: " + countPassengers(test2)); // Expected: 17

        // Test Case 3
        ArrayList<int[]> test3 = new ArrayList<>(Arrays.asList(new int[]{3, 0}, new int[]{9, 1}, new int[]{4, 8}, new int[]{12, 2}, new int[]{6, 1}, new int[]{7, 8}));
        System.out.println("Remaining passengers: " + countPassengers(test3)); // Expected: 21

        // Test Case 4
        ArrayList<int[]> test4 = new ArrayList<>(Arrays.asList(new int[]{0, 0}));
        System.out.println("Remaining passengers: " + countPassengers(test4)); // Expected: 0
        
        // Test Case 5
        ArrayList<int[]> test5 = new ArrayList<>(Arrays.asList(new int[]{100, 0}, new int[]{20, 30}, new int[]{5, 15}));
        System.out.println("Remaining passengers: " + countPassengers(test5)); // Expected: 80
    }
}

File: dataset/Vanilla/OPENAI_gpt-5-2025-08-07/Task35.c
#include <stdio.h>

int countPeople(const int stops[][2], int n) {
    int total = 0;
    for (int i = 0; i < n; ++i) {
        total += stops[i][0] - stops[i][1];
    }
    return total;
}

int main() {
    int test1[][2] = { {10,0}, {3,5}, {5,8} }; // Expected: 5
    int test2[][2] = { {3,0}, {9,1}, {4,10}, {12,2}, {6,1}, {7,10} }; // Expected: 17
    int test3[][2] = { {0,0} }; // Expected: 0
    int test4[][2] = { {5,0}, {5,5}, {10,10}, {1,1} }; // Expected: 0
    int test5[][2] = { {50,0}, {30,10}, {5,5}, {40,20}, {0,0}, {1,1} }; // Expected: 90

    printf("%d\n", countPeople(test1, sizeof(test1)/sizeof(test1[0])));
    printf("%d\n", countPeople(test2, sizeof(test2)/sizeof(test2[0])));
    printf("%d\n", countPeople(test3, sizeof(test3)/sizeof(test3[0])));
    printf("%d\n", countPeople(test4, sizeof(test4)/sizeof(test4[0])));
    printf("%d\n", countPeople(test5, sizeof(test5)/sizeof(test5[0])));

    return 0;
}